x-pack/docs/en/watcher/actions/email.asciidoc

```diff
@@ -349,9 +357,18 @@ xpack.notification.email.account:
             host: smtp-mail.outlook.com
             port: 587
             user: <email.address>
-            password: <password>
 --------------------------------------------------
 
+In order to not store the account SMTP password as plaintext in the
+elasticsearch yaml file, you should use the keystore instead (you can still
```

Comment:
We should crosslink to the keystore docs ("secure settings").

Reply:
done